README.md

```diff
@@ -30,8 +30,8 @@ The `createNav` function takes an object of options with the following propertie
 For example, to use the global-nav without the "login" section, and set the `max-width` to `80rem`:
 
 ``` html
-<script src="/node_modules/global-nav/dist/index.js"></script>
-<script>canonicalGlobalNav.createNav({showLogins: false, maxWidt: '80rem'});</script>
+<script src="/static/js/modules/global-nav.js"></script>
```

Comment:
That wont work for basic usage. Please, left this import as it is.

Reply:
what about the MaxWidt